Windows compilation error; `deno_permissions` is missing winapi feature flags

Reference: https://github.com/denoland/deno/blob/9092a23619c3b497342561e1c28444cf0308d9be/runtime/permissions/Cargo.toml#L27

The `processenv`, `wincon`, and `wincontypes` flags appear to be missing -> Several modules used are feature gated in the version of winapi deno is using

```
error[E0432]: unresolved import `winapi::um::processenv`
   --> C:\Users\user\.cargo\registry\src\index.crates.io-6f17d22bba15001f\deno_permissions-0.16.0\prompter.rs:159:19
    |
159 |   use winapi::um::processenv::GetStdHandle;
    |                   ^^^^^^^^^^ could not find `processenv` in `um`
    |
note: found an item that was configured out
   --> C:\Users\user\.cargo\registry\src\index.crates.io-6f17d22bba15001f\winapi-0.3.9\src\um\mod.rs:190:40
    |
190 | #[cfg(feature = "processenv")] pub mod processenv;
    |                                        ^^^^^^^^^^
    = note: the item is gated behind the `processenv` feature

error[E0432]: unresolved import `winapi::um::wincon`
   --> C:\Users\user\.cargo\registry\src\index.crates.io-6f17d22bba15001f\deno_permissions-0.16.0\prompter.rs:161:19
    |
161 |   use winapi::um::wincon::FlushConsoleInputBuffer;
    |                   ^^^^^^ could not find `wincon` in `um`
    |
note: found an item that was configured out
   --> C:\Users\user\.cargo\registry\src\index.crates.io-6f17d22bba15001f\winapi-0.3.9\src\um\mod.rs:266:36
    |
266 | #[cfg(feature = "wincon")] pub mod wincon;
    |                                    ^^^^^^
    = note: the item is gated behind the `wincon` feature

error[E0432]: unresolved import `winapi::um::wincon`
   --> C:\Users\user\.cargo\registry\src\index.crates.io-6f17d22bba15001f\deno_permissions-0.16.0\prompter.rs:162:19
    |
162 |   use winapi::um::wincon::PeekConsoleInputW;
    |                   ^^^^^^ could not find `wincon` in `um`
    |
note: found an item that was configured out
   --> C:\Users\user\.cargo\registry\src\index.crates.io-6f17d22bba15001f\winapi-0.3.9\src\um\mod.rs:266:36
    |
266 | #[cfg(feature = "wincon")] pub mod wincon;
    |                                    ^^^^^^
    = note: the item is gated behind the `wincon` feature

error[E0432]: unresolved import `winapi::um::wincon`
   --> C:\Users\user\.cargo\registry\src\index.crates.io-6f17d22bba15001f\deno_permissions-0.16.0\prompter.rs:163:19
    |
163 |   use winapi::um::wincon::WriteConsoleInputW;
    |                   ^^^^^^ could not find `wincon` in `um`
    |
note: found an item that was configured out
   --> C:\Users\user\.cargo\registry\src\index.crates.io-6f17d22bba15001f\winapi-0.3.9\src\um\mod.rs:266:36
    |
266 | #[cfg(feature = "wincon")] pub mod wincon;
    |                                    ^^^^^^
    = note: the item is gated behind the `wincon` feature

error[E0432]: unresolved import `winapi::um::wincontypes`
   --> C:\Users\user\.cargo\registry\src\index.crates.io-6f17d22bba15001f\deno_permissions-0.16.0\prompter.rs:164:19
    |
164 |   use winapi::um::wincontypes::INPUT_RECORD;
    |                   ^^^^^^^^^^^ could not find `wincontypes` in `um`
    |
note: found an item that was configured out
   --> C:\Users\user\.cargo\registry\src\index.crates.io-6f17d22bba15001f\winapi-0.3.9\src\um\mod.rs:267:41
    |
267 | #[cfg(feature = "wincontypes")] pub mod wincontypes;
    |                                         ^^^^^^^^^^^
    = note: the item is gated behind the `wincontypes` feature

error[E0432]: unresolved import `winapi::um::wincontypes`
   --> C:\Users\user\.cargo\registry\src\index.crates.io-6f17d22bba15001f\deno_permissions-0.16.0\prompter.rs:165:19
    |
165 |   use winapi::um::wincontypes::KEY_EVENT;
    |                   ^^^^^^^^^^^ could not find `wincontypes` in `um`
    |
note: found an item that was configured out
   --> C:\Users\user\.cargo\registry\src\index.crates.io-6f17d22bba15001f\winapi-0.3.9\src\um\mod.rs:267:41
    |
267 | #[cfg(feature = "wincontypes")] pub mod wincontypes;
    |                                         ^^^^^^^^^^^
    = note: the item is gated behind the `wincontypes` feature
 ```

--- a/runtime/permissions/Cargo.toml
+++ b/runtime/permissions/Cargo.toml
@@ -24,4 +24,4 @@ serde.workspace = true
 which = "4.2.5"
 
 [target.'cfg(windows)'.dependencies]
-winapi = { workspace = true, features = ["commapi", "knownfolders", "mswsock", "objbase", "psapi", "shlobj", "tlhelp32", "winbase", "winerror", "winuser", "winsock2"] }
+winapi = { workspace = true, features = ["commapi", "knownfolders", "mswsock", "objbase", "psapi", "shlobj", "tlhelp32", "winbase", "winerror", "winuser", "winsock2", "processenv", "wincon", "wincontypes"] }

